sdcm/nemesis.py

```diff
@@ -304,7 +304,16 @@ def _choose_file_for_destroy(self, ks_cfs):
             # For corruption we need to remove all files that their names are started from "mc-220-" (MC format)
             # Old format: "system-truncated-ka-" (system-truncated-ka-7-Data.db)
             # Search for "<digit>-" substring
-            file_name_template = re.search(r"(.*-\d+)-", file_name).group(1)
+
+            try:
+                file_name_template = re.search(r"(.*-\d+)-", file_name).group(1)
+            except Exception as error:  # pylint: disable=broad-except
+                self.log.debug('File name "{file_for_destroy}" is not as expected for Scylla data files. '
```

Comment:
file_for_destroy is empty string, do you want to output `file_name`?

Reply:
@juliayakovlev can you fix this log line ?

Reply:
fixed, thanks